plugins/legacy-plugin-chart-calendar/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+export function convertUTC(dttm: Date): Date {
+  return new Date(
+    dttm.getUTCFullYear(),
+    dttm.getUTCMonth(),
+    dttm.getUTCDate(),
+    dttm.getUTCHours(),
+    dttm.getUTCMinutes(),
+    dttm.getUTCSeconds(),
+  );
+}
+
+export function convertUTCTS(uts: number): number {
+  return convertUTC(new Date(uts)).getTime();
+}
+
+export function setUTC(d: Date): Date {
```

Comment:
nit: I would rather call it `getUTC` or `convertUTC` or similar as we're not mutating the original object.